src/sentry/integrations/repositories.py

```diff
@@ -112,7 +112,6 @@ def has_repo_access(self, repo: Repository) -> bool:
     def get_codeowner_file(self, repo: Repository, ref: str | None = None) -> Mapping[str, str]:
```

Comment:
We'll eventually have to update the return type to `Mapping[str, str] | None` so we should probably do it now.